src/iwant_bot/ignore.py

```diff
@@ -0,0 +1,23 @@
+import collections
+
+
+class IgnoreList(object):
+    def __init__(self):
+        self.ignored_users = collections.defaultdict(list)
```

Comment:
IgnoreList is a nicely designed class, there is no real need to expose `ignored_users` as a public attribute.